app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java

```diff
@@ -184,20 +184,35 @@ public float convertDpToPixel(float dp) {
 
     private void configureForOrientation(Configuration newConfig) {
         LinearLayout mainContainer = getView().findViewById(R.id.cover_fragment);
-        ViewGroup.LayoutParams params = imgvCover.getLayoutParams();
+        LinearLayout textContainer = getView().findViewById(R.id.cover_fragment_text_container);
+
+        LinearLayout.LayoutParams params = (LinearLayout.LayoutParams) imgvCover.getLayoutParams();
+        LinearLayout.LayoutParams textParams = (LinearLayout.LayoutParams) textContainer.getLayoutParams();
+
 
         if (newConfig.orientation == Configuration.ORIENTATION_PORTRAIT) {
+            double ratio = (float) newConfig.screenHeightDp / (float) newConfig.screenWidthDp;
+            double percentageWidth = 0.8;
+            if (ratio <= 1.7) {
+                percentageWidth = (ratio / 1.7) * percentageWidth * 0.8;
```

Comment:
Where does the 1.7 magic value come from? (16/9)? I would rather use something like `static final float SIXTEEN_BY_NINE = 1.8;` to make things more clear.

Reply:
fixed